src/SData.cpp

```diff
@@ -341,7 +341,7 @@ PVR_ERROR SData::GetEPGForChannel(ADDON_HANDLE handle, const PVR_CHANNEL &channe
 
         tag.iUniqueBroadcastId = event->uniqueBroadcastId;
         tag.strTitle = event->title.c_str();
-        tag.iChannelNumber = event->channelNumber;
+        tag.iUniqueChannelId = event->channelNumber;
```

Comment:
Not sure whether this is correct. Have you checked the actual semantics of `event->channelNumber`? This API change is not cosmetics, it's not just renaming a struct member. a channel unique id is semantically different form a channel number...